images/cert-manager/README.md

```diff
@@ -26,28 +26,41 @@ docker pull cgr.dev/chainguard/cert-manager:latest
 <!--getting:end-->
 
 <!--body:start-->
-## Using Cert Manager
-
-These set of images are a drop in replacement for the standard `cert-manager` installation ([here](https://cert-manager.io/docs/installation/)), and replacing them with the Chainguard images.
-
-For example, we can use these images with the helm installation and the following values:
-
-```yaml
-image:
-    repository: cgr.dev/chainguard/cert-manager-controller
-    tag: latest
-
-cainjector:
-    image:
-        repository: cgr.dev/chainguard/cert-manager-cainjector
-        tag: latest
-acmesolver:
-    image:
-        repository: cgr.dev/chainguard/cert-manager-acmesolver
-        tag: latest
-webhook:
-    image:
-        repository: cgr.dev/chainguard/cert-manager-webhook
-        tag: latest
+## Images
+Cert manager is composed of multiple container images. These can be deployed
+separately, but are typically deployed together leveraging the official
+[helm chart](https://cert-manager.io/docs/installation/helm/).
+
+The chainguard provided images for Cert manager are as follows:
+
+```bash
+cgr.dev/chainguard/cert-manager-controller:latest
+cgr.dev/chainguard/cert-manager-cainjector:latest
+cgr.dev/chainguard/cert-manager-acmesolver:latest
+cgr.dev/chainguard/cert-manager-webhook:latest
+```
+
+## Deploy using helm
+Please refer to the [upstream documentation](https://cert-manager.io/docs/installation/helm/)
+for full instructions on how to use and deploy Cert mananager via helm.
+
+Below is an example of how to use the Chainguard images with the helm chart:
+
+```bash
+# Refer to the helm chart docs and replace --version accordingly.
+helm install \
+  cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --create-namespace \
+    --version v1.13.3 \
+    --set image.repository=cgr.dev/chainguard/cert-manager-controller \
+    --set image.tag=latest \
+    --set cainjector.image.repository=cgr.dev/chainguard/cert-manager-cainjector \
+    --set cainjector.image.tag=latest \
+    --set acmesolver.image.repository=cgr.dev/chainguard/cert-manager-acmesolver \
+    --set acmesolver.image.tag=latest \
+    --set webhook.image.repository=cgr.dev/chainguard/cert-manager-webhook \
+    --set webhook.image.tag=latest \
+    # --set installCRDs=true # Optional, but chart installation will fail if CRDs are missing
```

Comment:
nit. If you leave this commented out, will the previous L63 work with the trailing `\`? Seems like it will expect a continuation, but not absolutely sure. <time passes...> tried it out, seems to fail, but maybe because I don't have things installed properly, running old helm, etc.

```
    # --set installCRDs=true # Optional, but chart installation will fail if CRDs are missing
Error: INSTALLATION FAILED: expected at most two arguments, unexpected arguments: #, #, Optional,, but, chart, installation, will, fail, if, CRDs, are, missing
```

Reply:
And then if remove it and remove the trailing `\`, I do get the expected error:
```
vaikas@vaikas-MBP enterprise-packages % helm install \
  cert-manager jetstack/cert-manager \
    --namespace cert-manager \
    --create-namespace \
    --version v1.13.3 \
    --set image.repository=cgr.dev/chainguard/cert-manager-controller \
    --set image.tag=latest \
    --set cainjector.image.repository=cgr.dev/chainguard/cert-manager-cainjector \
    --set cainjector.image.tag=latest \
    --set acmesolver.image.repository=cgr.dev/chainguard/cert-manager-acmesolver \
    --set acmesolver.image.tag=latest \
    --set webhook.image.repository=cgr.dev/chainguard/cert-manager-webhook \
    --set webhook.image.tag=latest

Error: INSTALLATION FAILED: failed to download "jetstack/cert-manager" at version "v1.13.3"
```

Reply:
Valid call, the code comment was an after-thought, but wasn't meant to break the ability to copy/paste the example.I've amended in a subsequent commit:

```
% helm install \
  cert-manager jetstack/cert-manager \
    --namespace cert-manager \
    --create-namespace \
    --version v1.13.3 \
    --set image.repository=cgr.dev/chainguard/cert-manager-controller \
    --set image.tag=latest \
    --set cainjector.image.repository=cgr.dev/chainguard/cert-manager-cainjector \
    --set cainjector.image.tag=latest \
    --set acmesolver.image.repository=cgr.dev/chainguard/cert-manager-acmesolver \
    --set acmesolver.image.tag=latest \
    --set webhook.image.repository=cgr.dev/chainguard/cert-manager-webhook \
    --set webhook.image.tag=latest \
    --set installCRDs=true
NAME: cert-manager
LAST DEPLOYED: Thu Jan  4 03:18:52 2024
NAMESPACE: cert-manager
STATUS: deployed
REVISION: 1
TEST SUITE: None
```